integration_tests/client/metrics/test_segmentation.py

```diff
@@ -107,9 +116,7 @@ def test_evaluate_segmentation_with_filter(
     color = Datum.metadata["color"]
     eval_job = model.evaluate_segmentation(
         dataset,
-        filter_by=[
-            color == "red",
-        ],
+        filters=Filter(datums=(color == "red")),  # type: ignore #605
```

Comment:
does this need to introduce a new type error?

Reply:
typo, this should be `issue #605`